userspace/libsinsp/parsers.cpp

```diff
@@ -4910,6 +4916,28 @@ void sinsp_parser::parse_setresuid_exit(sinsp_evt *evt)
 	}
 }
 
+void sinsp_parser::parse_setreuid_exit(sinsp_evt *evt)
+{
+	int64_t retval;
+
+	//
+	// Extract the return value
+	//
+	retval = evt->get_param(0)->as<int64_t>();
+
+	if(retval >= 0)
+	{
+		uint32_t new_euid = evt->get_param(1)->as<uint32_t>();
+
+		if(new_euid < std::numeric_limits<uint32_t>::max())
+		{
+			if (evt->get_thread_info()) {
```

Comment:
i would avoid the double call to `evt->get_thread_info()` and i would save it in a variable